doc/method/results.md

```diff
@@ -87,17 +72,13 @@ combination of the inputs has greater coverage than CoL, and in part
 because OTT has many names that are either not valid or not currently
 accepted.
 
-Since the GBIF source we used includes the 2011 edition of CoL [2011],
-OTT includes everything in that edition of CoL.  [did it get updated
+Since the GBIF source we used includes the 2011 edition of CoL [JAR: 2011],
+OTT includes everything in that edition of CoL.  [JAR: did it get updated
 for 2016 GBIF?]
```

Comment:
yes it did - latest CoL has the Ruggiero upper taxonomy, and GBIF has it as well.  But actually I'm not sure the recentness is material here... will review